frontend/hub/approvals/hooks/useApprovalActions.tsx

```diff
@@ -57,7 +57,8 @@ export function useApprovalActions(callback?: (collections: CollectionVersionSea
         onClick: (collection) => approveCollections([collection]),
         isDanger: false,
         isDisabled: (collection) =>
-          can_upload_signatures && require_upload_signatures && collection.is_signed
+          (can_upload_signatures && require_upload_signatures && collection.is_signed) ||
+          (collection.repository?.name !== 'staging' && collection?.repository?.name !== 'rejected')
```

Comment:
for staging, this is not correct, we can't use the names, it has to be the pipeline label

anything with `pipeline: staging` is also a staging repo

(there is no `pipeline: rejected`, so rejected is fine by name)

Reply:
This should be cleaned up now. :)